scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -194,9 +214,16 @@ task CreateImportTsvs {
       export TMPDIR=/tmp
 
       export GATK_LOCAL_JAR=~{default="/root/gatk.jar" gatk_override}
+      ~{for_testing_only}
 
+      if [ ~{has_service_account_file} = 'true' ]; then
+        gcloud auth activate-service-account --key-file='~{service_account_json}'
+        gsutil cp ~{input_vcf} .
+        gsutil cp ~{input_vcf_index} .
+      fi
+      
       gatk --java-options "-Xmx7000m" CreateVariantIngestFiles \
-        -V ~{input_vcf} \
+        -V ~{local_input_vcf} \
```

Comment:
have you tested this in Terra for a non-AOU/service account upload?

Reply:
i forgot to push my latest change, which should work when not passing in a service account. i do still need to test that.